src/commands/log.js

```diff
@@ -0,0 +1,51 @@
+import graphQLFetcher from '../util/graphQLFetcher'
+import getServiceBaseURL, {GAME} from '../util/getServiceBaseURL'
+import loadCommand from '../util/loadCommand'
+import parseArgvAndInvoke from '../util/parseArgvAndInvoke'
+
+export const {parse, usage, commandDescriptor} = loadCommand('log')
+
+function invokeResponseAPI(lgJWT, questionNumber, responseParams) {
+  const mutation = {
+    query: `
+mutation($response: CLISurveyResponse!) {
+  saveRetrospectiveCLISurveyResponse(response: $response) {
+    createdIds
+  }
+}
+    `,
+    variables: {response: {questionNumber, responseParams}},
+  }
+  return graphQLFetcher(lgJWT, getServiceBaseURL(GAME))(mutation)
+    .then(data => data.saveRetrospectiveCLISurveyResponse)
```

Comment:
This doesn't seem to be handling GraphQL errors anywhere. So if you enter a command that doesn't parse right on the server you don't see the error.